### PR TITLE
Send raw values when using WebChannel

### PR DESCRIPTION
**Pull request by thebrianchen**
As requested by @yuchenshi

### CODE REVIEW

File: packages/firestore/src/remote/serializer.ts [hunk 1]
```diff
@@ -167,12 +167,13 @@ export class JsonProtoSerializer {
    * our generated proto interfaces say Int32Value must be. But GRPC actually
    * expects a { value: <number> } struct.
    */
-  private toInt32Value(val: number | null): number | undefined {
-    if (!typeUtils.isNullOrUndefined(val)) {
+  private toInt32Value(val: number | null): number | null {
```
Comment (mikelehen) [inline packages/firestore/src/remote/serializer.ts]: You changed the return type (to `null`) but need to update the comment to match.

Comment (thebrianchen) [reply packages/firestore/src/remote/serializer.ts]: oops, good catch!

**Comment by yuchenshi** [comment]
I think we can totally test this against the old Firestore emulator v1.7.1: https://github.com/firebase/firebase-tools/commit/b1ee5c29001dc498a56556638e52cdfc37d910d7#diff-e1b9303f298bab6ec7fe10ee0fad137bR43

**Comment by yuchenshi** [comment]
Talked offline. I'll do a manual test and get back to you

**Pull request merged**